examples/api/python/pythonic/finite_field.py

```diff
@@ -0,0 +1,11 @@
+from cvc5.pythonic import *
+
+if __name__ == '__main__':
+    F = FiniteFieldSort(5)
+    a, b = FiniteFieldElems("a b", F)
+
+    solve(a * b == 1, a == 2)
```

Comment:
This does not fully correspond to `examples/api/python/finite_field.py`, it is missing assertions about the check-sat result and the `getValue` queries/printing.